### PR TITLE
chore(www): remove styled system Box component from guidelines system components altogether

### PR DESCRIPTION
**Pull request by abhishekjakhar**
## Description

The styled system based `<Box/>` component imported from **guidelines/system/box** was being used in all guidelines pages, but now all guidelines pages are using `<Box/>` component exported from theme-ui. So, the styled system based `<Box/>` is not getting used anywhere in codebase which makes it safe to remove.

### CODE REVIEW

File: www/src/components/guidelines/system/text.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from "@emotion/styled"
 import shouldForwardProp from "@styled-system/should-forward-prop"
```
Comment (fk) [inline www/src/components/guidelines/system/text.js:2]: LGTM, but would be nice to get these `@styled-system` things out there, plus ideally `styled`. Would you be up to help with that, too?

Comment (abhishekjakhar) [reply www/src/components/guidelines/system/text.js:2]: @fk yeah, I am working on it, I will raise the PR for all the components of styled-system used inside guidelines in one go or do you want me to do those changes in this same PR?

Comment (fk) [reply www/src/components/guidelines/system/text.js:2]: 🙏

Comment (abhishekjakhar) [reply www/src/components/guidelines/system/text.js:2]: I could not get time to complete those changes so I thought let’s push these changes first and will do it in second go

Comment (fk) [reply www/src/components/guidelines/system/text.js:2]: > do you want me to do those changes in this same PR?

However you want to do it;
lemme approve this one now!

Comment (abhishekjakhar) [reply www/src/components/guidelines/system/text.js:2]: Thank you very much 🙏

Comment (abhishekjakhar) [reply www/src/components/guidelines/system/text.js:2]: > > do you want me to do those changes in this same PR?
> 
> However you want to do it;
> lemme approve this one now!

Yes, I prefer doing it in second go

**Pull request merged**